### ISSUE TITLE
预测分数为何从这6个索引处取值？

### ISSUE THREAD

**Issue by quyichen** [taishan1994/prompt_text_classification#7]
pred1 = logits[i, mask1][[1071, 1599, 2650, 1328, 2699, 7770]]
pred2 = logits[i, mask2][[800, 1962, 839, 2626, 2584, 1069]]

**Comment by taishan1994** [comment]
> pred1 = logits[i, mask1][[1071, 1599, 2650, 1328, 2699, 7770]] pred2 = logits[i, mask2][[800, 1962, 839, 2626, 2584, 1069]]

标签的首尾的字在vocab.txt里面的索引。

**Comment by quyichen** [comment]
好的，谢谢

**Comment by Lord-Rhongomyniad** [comment]
您好，我想问下这些索引值是如何得到的。我通过with open结构打开文件并用.read()方法读入后，shiyong.index() 方法得到的索引值和项目中的值不一致
